recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = "1.27.28" %}
-{% set hash = "b00da6290d8d42c6c3048b045d6537421e8b83e03b4f24bb44c5ea9f37ab6258" %}
+{% set version = "1.27.59" %}
```

Comment:
Could we update to the latest **v1.28.2**? The requirements are the same https://github.com/boto/botocore/blob/1.28.2/setup.cfg

Reply:
`aiobotocore` needs this version.

https://github.com/aio-libs/aiobotocore/blob/2.4.0/setup.py